src/cocoa/toga_cocoa/widgets/webview.py

```diff
@@ -1,6 +1,7 @@
 from rubicon.objc import objc_method
 from travertino.size import at_least
 
+from toga_cocoa.keys import toga_key
 from toga_cocoa.libs import *
```

Comment:
At column 1: (F403) 'from toga_cocoa.libs import *' used; unable to detect undefined names